### ISSUE TITLE
Ability to widen the screen

### ISSUE THREAD

**Issue by Ashdemai** [ranfdev/Geopard#37]
Right now Geopard only uses a slim realestate to show the page with no ability to add/change in the configuration file. There are some text that will make you scroll horizontally when you can just show it if it was wide enough.

**Comment by earboxer** [comment]
Yeah, it seems like there's currently something wrong with Geopard:

![Image](https://github.com/user-attachments/assets/6cdad9b6-bde7-4f4d-b87e-07f7b0295453)

This is version 1.6.0 on alpine edge

**Comment by ranfdev** [comment]
I think you have this issue because of a gtk or libadwaita update. I've seen a similar thing while updating geopard to the gnome 48 flatpak SDK.

Anyway, I will make an updated release fixing this
